packages/consensus-validator-set-static/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "@mainsail/consensus-validator-set-static",
```

Comment:
Rename to @mainsail/validator-set-static. There is no need for consensus prefix.